.ddev/docker-compose.cypress.yaml

```diff
@@ -12,6 +12,8 @@ services:
       # Pass in the admin details from CI environments
       - CYPRESS_ADMIN_USERNAME
       - CYPRESS_ADMIN_PASSWORD
+      # For percy
+      - PERCY_TOKEN
```

Comment:
What would happen if `PERCY_TOKEN` is not available locally? I know that Docker compose won't throw an error but will the regular Cypress commands work fine?

Reply:
Yes, Iy will log an error message and continue with cypress run.

<img width="1027" alt="image" src="https://github.com/contrib-tracker/backend/assets/132985988/09252d09-f1e3-4f65-a728-72a956badbda">